easybuild/easyconfigs/f/fgbio/fgbio-1.3.0.eb

```
@@ -22,21 +22,21 @@ cmds_map = [(
     '.*',
     'sed -i \'s/scala.sys.process.Process("git rev-parse --short HEAD").lineStream.head/"%(version)s"/\' version.sbt'
     ' && sbt --sbt-dir "$TMPDIR/sbt" assembly --ivy "$TMPDIR/ivy2"'
-    ' && echo -e \'#!/bin/bash\\ncd $(dirname $0) && exec java -jar ../lib/fgbio-%(version)s.jar "$@"\' >> fbgio'
-    ' && chmod +x fbgio'
+    ' && echo -e \'#!/bin/bash\\nexec java -jar "${EBROOTFGBIO}/lib/fgbio-%(version)s.jar" "$@"\' >> fgbio'
+    ' && chmod +x fgbio'
```

Good catch. How about using templates to avoid this?
```suggestion
    ' && echo -e \'#!/bin/bash\\nexec java -jar "${EBROOTFGBIO}/lib/%(name)s-%(version)s.jar" "$@"\' >> %(name)s'
    ' && chmod +x %(name)s'
```

---

Makes sense, let me see...

---

Those templates also work on files_to_copy, sanity_check_paths and sanity_check_commands?

---

Commit added where I substituted the software name by its template.